src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -59,8 +63,29 @@ void Controller_RevitDocumentChanged(object sender, EventArgs e)
             PopulateItems();
         }
 
+        private void Updater_ElementsUpdated(object sender, ElementUpdateEventArgs e)
+        {
+            switch (e.Operation)
+            {
+                case ElementUpdateEventArgs.UpdateType.Added:
```

Comment:
Expect case  ElementUpdateEventArgs.UpdateType.Modified, other cases can all be the default? Should them be list here?

Reply:
Yes, other case may be useless here, I will update soon.